inference-engine/src/gna_plugin/gna_graph_tools.hpp

```diff
@@ -779,4 +779,22 @@ inline void CNNNetworkReconnectLayer(CNNLayerPtr old_prev_layer, CNNLayerPtr new
     }
 }
 
+/**
+ * @brief returns order of dimensions corresponding to NCHW layout for the given layout
+ * @param layout
+ */
+inline std::vector<std::size_t> GetFromIRDimsOrderNCHW(InferenceEngine::Layout layout) {
```

Comment:
Beside moving this function, I would refactor it (together with `FROM_IR_DIM`)
to make their usage more readable.

Reply:
GetDataDimSize() fumctions are introduced